### PR TITLE
Added 8 profile links

### PR DESCRIPTION
**Pull request by saarthakchats**

### CODE REVIEW

**Comment by saarthakchats** [comment]
#71

**Review by samyakbvs — APPROVED** [review]

**Comment by samyakbvs** [comment]
Great work! Saarthak

File: index.html [hunk 1]
```diff
@@ -749,9 +753,9 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="https://twitter.com/simon_riggs"><i class="icon social_twitter"></i></a>
-											<a target="default" href="https://2ndquadrant.com"><i class="icon fa fa-external-link"></i></a>
-
+											<a target="default" href="https://2ndquadrant.com"><i class="icon fa fa-external-link">
 											<a target="default" href="https://uk.linkedin.com/in/simonat2ndquadrantdotcom"><i class="icon social_linkedin"></i></a>
+												<a href="https://github.com/ppaeps"><i class="icon social_github"></i></a>
```
Comment (0x48piraj) [inline index.html]: Indent well

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
See the code reviews. Pay attention to indentation.

**Review by the-codingninja — APPROVED** [review]

**Comment by mariobehling** [comment]
It looks like this PR is on of the PRs that resulted in breaking the site. I have to revert it.

**Pull request merged**